projects/1.21/assets/ars-nouveau/ars_nouveau/lang/zh_cn.json

```
@@ -0,0 +1,2009 @@
+  "ars_nouveau.page.upgrades": "升级",
+  "ars_nouveau.page.vitalic_sourcelink": "利用附近的生物死亡和动物繁殖生成中等量的魔源。另外，生死魔源通道还会利用附近的幼年生物常态产出魔源，同时加速它们的成长。",
+  "ars_nouveau.page.void_jar": "捡起物品时会摧毁物品，并返还少量魔力。潜行使用湮灭之瓶可开启或关闭摧毁功能。副手持物品时使用湮灭之瓶，或对放有湮灭之瓶的抄写台使用物品，即可将其摧毁。湮灭之瓶只会在处于快捷栏时运作。",
+  "ars_nouveau.page.void_prism": "摧毁经过它的法术弹射物。",
+  "ars_nouveau.page.wand": "可在抄写台中向巫师之杖抄入单个法术。杖中法术会以“弹射物 -> 加速”起始，抄入的法术不可具有形态。可借此突破10个构件的限制。如果需要巫师之杖施放“破坏”，抄入的法术应当只有“破坏”，所得法术则是“弹射物 -> 加速 -> 破坏”。",
+  "ars_nouveau.page.warp_portal": "折跃传送门",
+  "ars_nouveau.page.warp_scroll": "一次性使用的卷轴，使用后可以传送到其中记录的位置，但无法跨维度传送。在副手持有已绑定的折跃卷轴，再对其他实体施放“闪现”，即可折跃该实体。",
+  "ars_nouveau.page.weald_walker": "林地行者",
+  "ars_nouveau.page.wilden": "荒野族裔",
+  "ars_nouveau.page.wixie_hat": "可让星宝石兽搬运药水。为星宝石兽戴上薇克精帽子后，使用支配之杖连接它们和药水罐即可。",
+  "ars_nouveau.page.world_generation": "世界生成",
+  "ars_nouveau.page1.alakarkinos_charm": "探宝蟹能筛滤沙子和沙砾，从中找出考古遗迹里会出现的物品。它们出现于沙滩，给予它们任意陶片后会回赠信物。",
+  "ars_nouveau.page1.allow_scroll": "可为自动化相关实体提供物品列表。将其放置抄写台上，再潜行对其使用物品或方块，即可将该物品或方块抄入卷轴。将卷轴交予星宝石兽，它就只会捡起或拿取卷轴中的物品。",
+  "ars_nouveau.page1.alteration_table": "用于将纤维编入魔法盔甲。使用改衣台前需先将盔甲放到台上。台上面板会显示盔甲可用的纤维槽。对面板使用纤维即可添加或移除。取走盔甲即会应用纤维的能力。移除纤维时，应先把盔甲放到台上，并以空手与面板交互。",
+  "ars_nouveau.page1.amethyst_golem_charm": "紫晶傀儡能采集、培育、拾取其家园附近的紫水晶。在紫水晶母岩旁施展启蒙仪式，即可获得紫晶傀儡护符。",
+  "ars_nouveau.page1.apparatus_crafting": "附魔装置是合成新生魔艺中用到的特殊机器、新饰品、新装备之所需。附魔装置的配方需要最多8个奥术基座、1个奥术核心，以及附魔装置自身。准备好所有设备后，应当以合成法师之花种子作为起始。",
+  "ars_nouveau.page1.applying_perks": "纤维可通过改衣台编织进盔甲中。每一件盔甲的纤维槽配置都不同，无论是数量还是等级。某些纤维有最低等级需求，不过大多数纤维都只是会根据所处槽位的等级增强能力。",
+  "ars_nouveau.page1.archwood": "至高木树有4种变种，均有小概率生成于任意生物群系。你可能偶尔会遇见一座至高木森林——满是魔法生物、自然出现的魔光、至高木树的森林。至高木可用于装饰、仪式，及各类法杖的制作。至高木树上有概率结出魔法果实；这些果实可以食用，也可以用来酿造药水。",
+  "ars_nouveau.page1.archwood_grate": "栅板上方的流体会向下渗漏，如可行还会使其下方的方块含水。与栅板交互即是与其下方的方块交互，可借此装起其下方块的流体。物品和弹射物会直接穿过栅板。可向任意方向放置。",
+  "ars_nouveau.page1.armor": "魔法衣物能增加穿戴者的魔力恢复速度，且可借助纤维获得特殊能力。术士套装的防御能力最低，但它拥有最强力的纤维槽组。秘术师和战法师套装的防御能力均较高，但它们的纤维槽较少且较弱。更多与纤维有关的信息请参阅盔甲与能力的有关条目。",
+  "ars_nouveau.page1.armor_upgrading": "魔法盔甲可使用纤维升级，以获得额外效果。每种盔甲的纤维槽数目不同，纤维槽的等级也不同。槽位的等级越高，纤维的能力就越强，部分纤维更是要求有最低等级需求。纤维的编织方法请参阅改衣台条目。",
+  "ars_nouveau.page1.basic_spell_turret": "给予炮塔红石信号可让其施法，和发射器类似。炮塔接受使用“触摸”或“弹射物”的法术。使用抄有法术的法术羊皮纸可设置法术。炮塔的施法需要从附近魔源罐抽取魔源。若炮塔有容器相邻，则其可使用“拾取”和“放置方块”。",
+  "ars_nouveau.page1.better_casting": "你的魔力池可通过特殊魔法盔甲、魔咒、习得新魔符、饮用药水等方法扩充。获得法师之花种子之后，即可合成许多种可自由配置的魔法盔甲，再使用纤维加以调整。",
+  "ars_nouveau.page1.bookwyrm_charm": "书龙护符可对存储讲台使用，以增加其可用容器上限。以书与笔增强启蒙仪式即可获得护符。书龙可以染成白色、黑色、蓝色、绿色、红色、紫色。",
+  "ars_nouveau.page1.decorative": "纯粹的装饰性方块。可向切石机放入魔源石以查看完整列表。",
+  "ars_nouveau.page1.deny_scroll": "可为自动化相关实体提供物品列表。将其放置抄写台上，再潜行对其使用物品或方块，即可将该物品或方块抄入卷轴。将卷轴交予星宝石兽，它就只会捡起或拿取不在卷轴中的物品。",
+  "ars_nouveau.page1.drygmy_charm": "德格米通常会跟随和照料其周围的动物。它们会出现在世界各处，虽然不算常见。可以在世界中为德格米指定家园，它们会产出附近怪物和动物的击杀掉落物，而不会实际伤害它们。可以在野生的德格米附近丢下荒野之角以表示友好！德格米可以染成青色、橙色、棕色。",
+  "ars_nouveau.page1.enchanters_fishing_rod": "可在抄写台上向巫师钓鱼竿抄入单个法术。钓鱼竿的法术会以“触摸”起始，抄入的法术不可具有形态。巫师钓鱼竿可像普通的钓鱼竿一样使用；但在勾住实体后，持有者还可向该实体施法，直至收回渔线。",

```

```suggestion
  "ars_nouveau.page1.enchanters_fishing_rod": "可在抄写台上向巫师钓鱼竿抄入单个法术。钓鱼竿的法术会以“触摸”起始，抄入的法术不可具有形态。巫师钓鱼竿可像普通的钓鱼竿一样使用；但在勾住实体后，持有者还可向该实体施法，直至渔线断开。",

```
这里是渔线断开吧。不过MC原版关于钓鱼，[钓鱼竿的音效](https://zh.minecraft.wiki/w/%E9%92%93%E9%B1%BC%E7%AB%BF#%E9%9F%B3%E6%95%88)只有关于浮漂的。真要说收回和断开有什么不同，经测试，应该是钩住实体，收回操作会带动实体移动，但是断开渔线不会带动实体移动。
测试方法：用钓鱼竿钩住物品，分别进行右击收回浮漂（渔线）与保持渔线与物品钩住然后角色迅速移动，直至渔线消失。
测试结果：右击收回浮漂带动了物品移动；持渔线与物品钩住然后角色迅速移动，直至渔线消失，物品并未产生移动。